chromium_src/ui/color/color_mixers.cc

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "ui/color/color_mixers.h"
+
+#include "ui/base/ui_base_features.h"
+
+namespace {
+
+bool GiveTrue() {
+  return true;
+}
+
+}  // namespace
+
+// AddMaterialUiColorMixer() should be called to get some colors(ex,
+// kColorToastBackground) at specific order in the AddColorMixers().
+// See the comments at ui::AddColorMixers().
+#define IsChromeRefresh2023 IsChromeRefresh2023() || GiveTrue
```

Comment:
@simonhong, do you want to get rid of the change from https://github.com/brave/brave-core/pull/22320/commits/b39c27459f5c34d61a95019547ef676f8dc70e38 that calls `AddMaterialSidePanelColorMixer` from `browser/ui/color/brave_color_mixer.cc` and move the color changes  to the `browser/ui/color/brave_material_side_panel_color_mixer.cc` in here?

Reply:
Ah, never mind, that on is actually called from `chrome/browser/ui/color/chrome_color_mixers.cc`, but I think we could move the colors here still?

Reply:
Yeah, addressed - https://github.com/brave/brave-core/pull/22996/commits/f8bd1ecee9caedfde632ccab3625f7e11bc27826